app/views/admin/source_files/_index_header.html.erb

```diff
@@ -7,6 +7,22 @@
     <%= content_for(:title) %>
   </h1>
 
+  <div>
+    <% if params[:ready_for_redaction] %>
+      <%= link_to(
+            "Show all files",
+            admin_source_files_path,
+            class: "button"
+          ) %>
+  <% else %>
+    <%= link_to(
+          "Only show files ready for redaction",
+          admin_source_files_path(ready_for_redaction: true),
```

Comment:
What you have is totally fine but just FYI I found about this cool [`url_for`](https://apidock.com/rails/ActionView/RoutingUrlFor/url_for) helper recently and it's perfect for situations like this. It has a lot of different uses, but one of them is to just take the current URL and add or remove query params. So for example you could use it like this

```suggestion
          url_for(ready_for_redaction: true),
```

That way you don't have to repeat the path helper over and over when you just want to change the query params.

Again no need to change what you have here--I'm just sharing because I thought it was cool!

Reply:
@jutonz Thanks great! Thanks for sharing! I'll apply this